girder/utility/hash_state.py

```diff
@@ -25,6 +25,9 @@
 import hashlib
 import ctypes
 import binascii
+import sys
+
+_HASHLIB_INLINE_EVP_STRUCT = (sys.version_info < (2, 7, 13) or sys.version_info >= (3,))
```

Comment:
It looks like the commits to address [Python Issue #26470](https://bugs.python.org/issue26470) are in 2.7, 3.5, and 3.6, so I doubt `sys.version_info >= (3,)` is correct.